x-element.js

```diff
@@ -1643,106 +1466,422 @@ class TemplateEngine {
     }
   }
 
-  static #map(type, value, lastValue, { node, startNode }, { identify, callback }, name) {
-    if (type === 'content') {
-      if (Array.isArray(value)) {
-        const state = TemplateEngine.#setIfMissing(TemplateEngine.#stateMap, startNode, () => ({}));
-        TemplateEngine.#mapInner(state, node, startNode, identify, callback, value, name);
-      } else {
-        throw new Error(`Unexpected ${name} value "${value}".`);
+  static #map(node, startNode, value, identify, callback) {
+    TemplateEngine.#mapInner(node, startNode, identify, callback, value, 'map');
+  }
+
+  // Deprecated. Will remove in future release.
```

Comment:
The git diff is pretty ugly in here… but… I’ve purposefully set all the deprecated stuff up so that it’ll be a breeze to remove in a follow-on PR once we get to `2.x`.